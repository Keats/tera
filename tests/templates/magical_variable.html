{% import "macros.html" as macros %}

{{ __tera_context }}


{{ macros::dump(name="bob") }}
