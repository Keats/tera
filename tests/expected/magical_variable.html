

{
  "a_tuple": [
    1,
    2,
    3
  ],
  "an_array_of_tuple": [
    [
      1,
      2,
      3
    ],
    [
      1,
      2,
      3
    ]
  ],
  "empty": [],
  "friend_reviewed": true,
  "number_reviews": 2,
  "product": {
    "manufacturer": "Motorala",
    "name": "Moto G",
    "price": 100,
    "summary": "A phone"
  },
  "reviews": [
    {
      "paragraphs": [
        "A",
        "B",
        "C"
      ],
      "title": "My review"
    },
    {
      "paragraphs": [
        "A",
        "B",
        "C"
      ],
      "title": "My review"
    }
  ],
  "show_more": true,
  "username": "bob"
}


{
  "name": "bob"
}
